component/badges/badges.css

```diff
@@ -0,0 +1,92 @@
+.badge_container {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  border: 1px solid lightgray;
+  margin-top: 1rem;
+  padding: 1rem;
+}
+
+.badge {
+  padding: 0.3rem;
+  font-size: 0.75rem;
+  font-weight: bold;
+}
+.badge-primary {
+  background-color: var(--primary);
+  color: var(--text);
+}
+.badge-secondary {
+  border: 1px solid var(--dark);
+}
+.badge-dark {
+  background-color: var(--dark);
+  color: var(--text);
+}
+.badge-warning {
+  background-color: var(--warning);
+  color: var(--text);
+}
+.badge-danger {
+  background-color: var(--danger);
+  color: var(--text);
+}
+.badge-success {
+  background-color: var(--success);
+  color: var(--text);
+}
+.badge-pill {
+  border-radius: 10px;
+}
+.badge-link {
+  text-decoration: none;
```

Comment:
you can declare text-decoration: none; globally for all links as default and then change it where u need text-decoration

Reply:
all ready make it , but i don't want to make class name much bigger , that's why re-declared here